redis.go

```diff
@@ -238,3 +261,50 @@ func (c *RedisClient) getResponse(con redis.Conn, id string) (*Response, error)
 
 	return response, nil
 }
+
+// Stream listens to a stream of events from the server
+func (c *RedisClient) Stream(ctx context.Context, module string, object ObjectID, event string) (<-chan Event, error) {
+	con := c.pool.Get()
+	key := fmt.Sprintf("%s.%s.%s", module, object, event)
+	fmt.Println("subscribe to", key)
+	_, err := con.Do("SUBSCRIBE", key)
+
+	if err != nil {
+		con.Close()
+		return nil, err
+	}
+
+	ch := make(chan Event)
+	go func(con redis.Conn) {
+		defer func (){
+		    close(ch)
+		    con.Send("UNSUBSCRIBE")
+		    con.Close()		    
+		 }()
+		defer con.Send("UNSUBSCRIBE")
```

Comment:
Seems github didn't do the replacement properly. Lines 248,249 needs to go

Reply:
Good catch!
Fixed